recipes/go/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Move the go directory into $PREFIX/ so it can be built in its install location.
+cd .. && mv go $PREFIX/ && cd $PREFIX/go
+
+# Change to `src` build and then move out.
+cd src
+./all.bash
+cd ..
+
+# Delete `test` directory as it is no longer needed.
+rm -rf test
```

Comment:
When I had added this, I was mainly trying to cut down on the size of the final package.